src/components/SessionTokenRefresher.tsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect } from "react";
+import { useDispatch } from "react-redux";
+import { AppDispatch } from "store";
+
+import { refreshSessionToken } from "helpers/refreshSessionToken";
+
+const TOKEN_REFRESH_INTERVAL = 2 * 60_000; // 60,000ms = 1 minute
+
+export const SessionTokenRefresher: React.FC = () => {
+  const dispatch: AppDispatch = useDispatch();
+
+  useEffect(() => {
+    // Function to call refreshSessionToken every minute
+    const tickerId = setInterval(() => {
+      try {
+        refreshSessionToken(dispatch);
+      } catch (error) {
+        console.error("Error refreshing session token: ", error);
+      }
+    }, TOKEN_REFRESH_INTERVAL);
```

Comment:
The interval is set to 2 minutes, but the comment says 1 minute. Just making sure the intent is to check every 2 minutes instead of 1.

Reply:
Yeah, the intent is 2 minutes. I'll fix the comment, TY!